doc/ref_cert/lfn/chapters/chapter02.md

```diff
@@ -35,6 +36,40 @@ All Terms utilized throughout this chapter are intended to align with CVC defini
 ## 2.1 Introduction
 NFVI (Network Functions Virtualization Infrastructure) refers to the physical resources (compute, storage and network) on which virtual network functions (VNFs) are deployed. As such, the performance of a VNF depends on the underlying NFVI over which it is hosted. A certain VNF may perform good in one hardware and may perform worst in another. Thus, a need arises to certify NFVI that can help in onboarding VNFs onto a hardware with an acceptable level of VNF performance. Certain frameworks like [yardstick](https://github.com/opnfv/yardstick), [vsperf](https://github.com/opnfv/vswitchperf) etc. provide a set of tests that can be run on a hardware to obtain Key Performance Indicators (KPIs) which give a measurable output of the NFVI's performance. With these KPIs, a decision can be made on the VNFs that can offer an acceptable level of performance when on-boarded on the NFVI.
 
+<a name="2.1.1"></a>
+### 2.1.1 CNTT RI and RC toolchains
+
+[OPNFV](https://www.opnfv.org/) has built a complete CI/CD toolchain
+continuously deploying and testing NFVI. It's composed of centralized
+components de facto used by the CNTT RI hosted by OPNFV
+([CIRV](https://wiki.opnfv.org/pages/viewpage.action?pageId=47284396)).
+
+As for all OPNFV installer projects,
+[Jenkins](https://build.opnfv.org/ci/view/cntt/) triggers CIRV deployments,
+runs the OPNFV test cases part of the NFVI verification and then publishes all
+test results in the
+[centralized test database](https://docs.opnfv.org/en/stable-hunter/_images/OPNFV_testing_working_group.png)
+and all artifacts (reports, logs, etc.) to
+[an S3 compatible storage service](http://artifacts.opnfv.org/).
+
+The CNTT compliance and certification processes should benefit from the OPNFV
```

Comment:
@collivier - 

Consider re-writing this sentence as recommended below by removing CNTT references, and promoting existing tool-chain(s):

Current sentence:
 
"The CNTT compliance and certification processes should benefit from the OPNFV testing history and then must conform to this overall toolchain design without reinventing the wheel."

Recommended replacement sentence:

"The RC process will leverage existing OPNFV testing knowledge (projects) and experience (history) with the integration and augmentation of OPNFV/OVP certification tool-chains in place today."

Reply:
Done from my understanding. (slightly changed as OVP is about governance)
Thank you